### DISCUSSION TITLE
Welcome to zkWasm Discussions!

### DISCUSSION THREAD

**Discussion by xgaozoyoe** [DelphinusLab/zkWasm#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by em1nat0r** [comment]
hey there

In this [paper](https://jhc.sjtu.edu.cn/~hongfeifu/manuscriptb.pdf) page 17:
"As a case study, we compare the SHA256 execution trace with and without inline instructions in Table 12."
Is there a way to reproduce it on my computer and compare time difference?
@xgaozoyoe

**Comment by Berylhfm** [comment]
Appreciate for all the work team has already done. Looking forward to seeing all kinds of deployment and development with ZKWASM.
